server/api/role/list/controller.test.js

```diff
@@ -97,6 +112,12 @@ describe('api/v1/role.list', () => {
       .send();
 
     expect(res.status).toBe(200);
+    // NOTE: Even though the user is assigned 3 roles, one of them is global.
```

Comment:
This comment here doesn't make much sense to me as an external reader. It should be either simplified or removed altogether imo.

Reply:
eh, i seem to be really bad at explaining technical stuff over text /shrug .